code/modules/hydroponics/hydroitemdefines.dm

```diff
@@ -75,25 +75,65 @@
 
 /obj/item/weapon/hatchet
 	name = "hatchet"
-	desc = "A very sharp axe blade upon a short fibremetal handle. It has a long history of chopping things, but now it is used for chopping wood."
+	desc = "A very sharp axe blade upon a short handle. Commonly seen embedded in the skull of a dead assistant. Or chopping wood. Alt-Click it to set a warcry."
 	icon = 'icons/obj/weapons.dmi'
 	icon_state = "hatchet"
 	flags = CONDUCT
-	force = 12
+	force = 10
 	w_class = 1
 	throwforce = 15
+	embed_chance = 50
+	embedded_impact_pain_multiplier = 3
 	throw_speed = 3
-	throw_range = 4
+	throw_range = 6
 	materials = list(MAT_METAL = 15000)
-	origin_tech = "materials=2;combat=2"
-	attack_verb = list("chopped", "torn", "cut")
+	origin_tech = "materials=1;combat=1"
+	attack_verb = list("chopped", "scalped", "cut")
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	sharpness = IS_SHARP
+	var/war_cry = "WHOOP WHOOP WHOOP!!"
 
 /obj/item/weapon/hatchet/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] is chopping at [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	user.visible_message("<span class='suicide'>[user] is scalping [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to get closer to their ancestors!</span>")
+	user.say("[war_cry]")
 	playsound(loc, 'sound/weapons/bladeslice.ogg', 50, 1, -1)
 	return (BRUTELOSS)
+	
+/obj/item/weapon/hatchet/AltClick()

```

Comment:
is this properly accounting for adjacency and only living mobs handling this item?

i.e can you alt click this as a ghost and at distance